SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetrics.cpp

```diff
@@ -20,12 +23,14 @@ namespace SpatialMetricsPrivate
 enum class EServerCommands : uint8
 {
 	StartInsights,
+	StopInsights,
 
 	ServerCommandsCount,
 	ServerCommandInvalid = ServerCommandsCount,
 };
 
-const FString ServerCommandNames[static_cast<uint8>(EServerCommands::ServerCommandsCount) + 1] = { TEXT("StartInsights"), TEXT("Invalid") };
+const FString ServerCommandNames[static_cast<uint8>(EServerCommands::ServerCommandsCount) + 1] = { TEXT("StartInsights"),
```

Comment:
If you use an UENUM you get all this and Count for free.

Reply:
UENUM gives you enum >> string, but not string >> enum. Unless I'm mistaken?

Reply:
https://docs.unrealengine.com/4.26/en-US/API/Runtime/CoreUObject/UObject/UEnum/GetIndexByName/

Reply:
Hmm maybe trickier than I thought.